Userland/Libraries/LibJS/JIT/GDB-elf.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023, Jesús Lapastora <cyber.gsuscode@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJIT/ELFBuild.h>
+#include <LibJS/JIT/GDB.h>
+
+namespace JS::JIT {
+using namespace ::JIT;
+Optional<FixedArray<u8>> build_gdb_image(ReadonlyBytes code)
```

Comment:
Since this is mostly LibJS independent, I think this should be moved to LibJIT and take the symbol name as an argument.